src/lex/cpp.rs

```diff
@@ -1152,6 +1158,25 @@ impl<'a> PreProcessor<'a> {
         }
         Ok(ret)
     }
+
+    fn update_builtin_definitions(&mut self) {
+        use Definition::Object;
+        let int_def = |i: i32| Object(vec![LiteralToken::Int(RcStr::from(i.to_string())).into()]);
+        let str_def = |s: &str| {
+            let rcstr = RcStr::from(format!("\"{}\"", s));
+            Object(vec![LiteralToken::Str(vec![rcstr]).into()])
+        };
```

Comment:
I feel like I see these a lot, maybe we could make them functions so they can be reused?

Reply:
I'll leave that for another PR since we'd want to change it in all of the current places as well. This is a little bit different and more complex than usual because it wraps in `Definition::Object(vec![...])`.

Reply:
Never mind, I went ahead and changed it.